### ISSUE TITLE
Instructions for local runs of integration tests are unclear

### ISSUE THREAD

**Issue by dzelemba** [GoogleCloudPlatform/healthcare-dicom-dicomweb-adapter#80]
The default settings use restricted access projects like gcp-healthcare-oss-test and cloud-healthcare-containers which most folks won't have access to. There are also specific encrypted values that need to be decrypted via KMS so it's not clear if just using different projects would even work.

**Comment by pavertomato** [comment]
>The default settings use restricted access projects like gcp-healthcare-oss-test 

tests require creation of temp dicom stores, i think it's not quite safe to create project with such public availability, so tests have configurable dataset path

> There are also specific encrypted values that need to be decrypted via KMS so it's not clear if just using different projects would even work.

actually i've because of this i've used kms not in usual way like with other projects - cloud build accesses it only on stage of publishing, which is disabled by default, and decryption has been moved to script https://github.com/GoogleCloudPlatform/healthcare-dicom-dicomweb-adapter/blob/master/integration_test/scripts/publish-images.sh

so any user able to run it with own values of  _LOCATION _PROJECT _DATASET like how we do for our forks of repo

**Comment by dzelemba** [comment]
Yes I agree we shouldn't create a public project for this, I was just looking for more complete instructions (e.g. when trying to figure it out myself, first I got an error about SHORT_SHA missing, then about access to public project, than about KMS API not being turned in project at which point I gave up).

Ok can we update the instructions to give an example of a complete command that will work? Also a list of APIs that need be to enabled in the given project and a sample gcloud services enable command to turn them all on would make this much easier.
